terraform/src/index.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+
+const Layer0Cache = require('./layer0-cache').Layer0Cache;
+const zlib = require('zlib');
+const aws = require('aws-sdk');
+const ecs = new aws.ECS();
+const kinesis = new aws.Kinesis({
+    accessKeyId: process.env.STREAM_AWS_ACCESS_KEY,
+    secretAccessKey: process.env.STREAM_AWS_SECRET_KEY,
+    region: process.env.STREAM_AWS_REGION
+});
+
+const l0Cache = new Layer0Cache(ecs);
+
+exports.handler = function(event, context, callback) {
```

Comment:
I don't feel like I have much to say regarding this code that hasn't already been said... it makes enough sense to me, flow-wise, but I'm not proficient enough to node to make very meaningful critique.